src/ts/components/button.ts

```diff
@@ -1,15 +1,16 @@
 import {ComponentConfig, Component} from './component';
 import {DOM} from '../dom';
 import {EventDispatcher, NoArgs, Event} from '../eventdispatcher';
+import { LocalizableText , i18n} from '../localization/i18n';
```

Comment:
Add space in `{ … }`
```suggestion
import { LocalizableText , i18n } from '../localization/i18n';
```